src/project.tsx

```diff
@@ -1,8 +1,44 @@
+import { projects } from "./app.constants";
+import FullStop from "./components/ui/full-stop";
+import ProjectItem from "./components/ui/project-item";
+
 function Project() {
   return (
-    <section>
-      <h1>Project</h1>
-      <p>Project page</p>
+    <section className="project-section">
+      <div className="container">
+        <div className="header">
+          {" "}
+          <h1>Projects</h1> <FullStop />
+        </div>
+        <div className="project-box">
+          {projects.map((project, i) => (
+            <ProjectItem
+              title={project.title}
+              description={project.description}
+              githubLink={project.githubLink}
+              liveLink={project.liveLink}
+              img={project.img}
+              key={i}
+            />
+          ))}
+          {/* <div className="project">
+            <h2>Project 2</h2>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed
+              tincidunt, odio et tincidunt fermentum, metus sapien ornare nunc,
+              nec tempus tortor lacus a nulla.
+            </p>
+          </div>
+          <div className="project">
+            <h2>Project 3</h2>
+            <p>
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed
+              tincidunt, odio et tincidunt fermentum, metus sapien ornare nunc,
+              nec tempus tortor lacus a nulla.
+            </p>
+          </div> */}
```

Comment:
❓ why do you comment this out

Reply:
oooh
I will clear it now sir.

I forgot about it